### PR TITLE
feat(crux): add pino logger, and small config refactors

### PR DESCRIPTION
**Pull request by polaroi8d**
PR contains the followings: 
- Implemented Pino logger
- Moved a few files to the config folder
- Small code improvements

### CODE REVIEW

File: web/crux/src/config/pino.logger.config.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      transport:
+        // Use pino-pretty only in the development environment
+        process.env.NODE_ENV !== 'production'
+          ? {
+              target: 'pino-pretty',
+              options: {
+                singleLine: true,
+                ignore: 'pid,hostname',
+                translateTime: 'mm/dd/yyyy h:MM:ss',
+                messageFormat: '[{req.headers.host}{req.url}][{context}] {msg} [{responseTime} ms][{res.statusCode}]',
```
Comment (polaroi8d) [inline web/crux/src/config/pino.logger.config.ts:24]: In the long run, we should do this a better way.

File: web/crux/.env.example [hunk 2]
```diff
@@ -13,6 +13,7 @@ JWT_SECRET=jwt-secret-token
 # install script docker pull disabled for the e2e tests
 AGENT_INSTALL_SCRIPT_DISABLE_PULL=true
 
+# Possible values: trace, debug, info, warn, error, and fatal
 LOG_LEVEL=verbose
```
Comment (m8vago) [inline web/crux/.env.example:17]: how is verbose the default then?

File: web/crux/src/config/config.config.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const configModuleConfig = {
```
Comment (m8vago) [inline web/crux/src/config/config.config.ts:1]: The file name would be nicer just simply as `app.config.ts` IMHO.

File: web/crux/src/config/pino.logger.config.ts (hunk 1 above)
Comment (m8vago) [reply web/crux/src/config/pino.logger.config.ts:24]: Can we include the `req.method` as well?

**Review by m8vago — CHANGES_REQUESTED** [review]

File: web/crux/src/config/pino.logger.config.ts (hunk 1 above)
Comment (polaroi8d) [reply web/crux/src/config/pino.logger.config.ts:24]: Not now, we have to figure out the right format for these, but this is not the time 🙉

File: web/crux/src/config/config.config.ts (hunk 3 above)
Comment (polaroi8d) [reply web/crux/src/config/config.config.ts:1]: Yeah, good point. I'll rename the object too.

**Review by m8vago — APPROVED** [review]

File: web/crux/.env.example (hunk 2 above)
Comment (m8vago) [reply web/crux/.env.example:17]: Possible values: trace, debug, info, warn, error, and fatal
there is no verbose in that list

**Review by m8vago — CHANGES_REQUESTED** [review]
Right now it leaks the cookies into the logs. That needs to be resolved before merge.

File: web/crux/.env.example (hunk 2 above)
Comment (polaroi8d) [reply web/crux/.env.example:17]: Ahh 🦅

**Review by m8vago — APPROVED** [review]

**Review by m8vago — APPROVED** [review]

**Pull request merged**